src/test/components/GlobalTrack.test.js

```diff
@@ -0,0 +1,131 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+// @flow
+
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import {
+  changeSelectedThread,
+  hideGlobalTrack,
+} from '../../actions/profile-view';
+import GlobalTrack from '../../components/timeline/GlobalTrack';
+import {
+  getGlobalTracks,
+  getRightClickedTrack,
+} from '../../reducers/profile-view';
+import { getSelectedThreadIndex } from '../../reducers/url-state';
+import { getProfileWithNiceTracks } from '../fixtures/profiles/tracks';
+import { storeWithProfile } from '../fixtures/stores';
+import { getMouseEvent } from '../fixtures/utils';
+
+const LEFT_CLICK = 0;
+const RIGHT_CLICK = 2;
+
+describe('timeline/GlobalTrack', function() {
+  /**
+   *  getProfileWithNiceTracks() looks like: [
+   *    'show [thread GeckoMain process]',
+   *    'show [thread GeckoMain tab]',       <- use this global track.
+   *    '  - show [thread DOM Worker]',
+   *    '  - show [thread Style]',
+   *  ]
+   */
+  function setup() {
```

Comment:
nit: style-wise I prefer having helper functions at the bottom of the test so you can see more directly the actual tests when opening a file. But this is very dependent on what you are doing already in the project (and personal preferences :) )

Reply:
Thanks, we have been doing helpers inline with the describe blocks like so. Good to know what others preferences are though.